Sources/Hummingbird/Application.swift

```diff
@@ -232,14 +232,14 @@ public struct HBApplication<Responder: HBResponder, ChildChannel: HBChildChannel
     ///   - onServerRunning: Function called once the server is running
     ///   - eventLoopGroupProvider: Where to get our EventLoopGroup
     ///   - logger: Logger application uses
-    public init<Context>(
-        router: HBRouter<Context>,
+    public init<ResponderBuilder: HBResponderBuilder>(
+        router: ResponderBuilder,
```

Comment:
The only remark is that I don't know how correct this argument's name is

Reply:
Yeah maybe another to add to #343

The idea of adding the router parameter was to make it clear we were adding a router but in theory it could be anything that creates a responder.